src/reqstool/common/validators/semantic_validator.py

```diff
@@ -173,12 +173,8 @@ def _validate_annotation_tests_refers_to_existing_svc_ids(
     ) -> List[ValidationError]:
         errors: List[ValidationError] = []
         for model, model_data in combined_raw_dataset.raw_datasets.items():
-            # Continue if model is not inital_urn
-            if (
-                model is not combined_raw_dataset.initial_model_urn
-                or not model_data.annotations_data
-                or not model_data.annotations_data.tests
-            ):
+            # Continue if no annotations data
+            if not model_data.annotations_data:
```

Comment:
shouldn't `or not model_data.annotations_data.tests` be added to the if statement also? If there are not annotated tests then there is nothing to compare the annotations towards?

Reply:
Not saying that it has to be there, it just seems appropriate?

Reply:
Yes i could add it, tested now and the result is the same.
I guess it checks the same thing in a way, if there are no annotations there are no tests as the tests are collected from the annotations?

Reply:
Yes I get it, It's the discussion about what came first, the egg or the chicken. I'll approve.